images/cloud-event-proxy.yml

```diff
@@ -23,10 +23,7 @@ from:
   builder:
   - stream: golang
   member: openshift-enterprise-base
-# note that errata config went to the wrong repo for this at 4.11 and 4.12 GAs, which broke not only
-# the main release but another non-ART component that also depends on it being here. so don't change
-# this, change errata to match this if it is necessary:
```

Comment:
i left this note. at the time we did not have name_in_bundle to override the convention, so we had to either change errata config or change the bundle image-references upstream, either way painful. since we do have that knob now, and it's evident we're never going to get SP to get the CDN repo right on the first round, might as well go with the flow and match what they have.